### ISSUE TITLE
The model is not open source

### ISSUE THREAD

**Issue by koute** [paperswithcode/galai#9]
On [the website](https://galactica.org/mission/) it says:

> We believe models want to be free and so we open source the model for those who want to extend it.

And [in releases](https://github.com/paperswithcode/galai/releases/tag/1.0.0) it says:

> We believe models should be open
> 
> Just as open source code has accelerated technological progress, so will open source machine learning research.
> 
> We built Papers with Code four years ago on the principle that things are best done in the open.
> 
> We stick to this pledge by open sourcing our model, and we will look to work to improve accessibility in the coming months.

But your model is released [under CC-BY-NC](https://github.com/paperswithcode/galai/blob/main/LICENSE-MODEL.md) which is not free ([according to FSF's definition](https://www.gnu.org/licenses/license-list.html#CC-BY-NC)) nor open source ([according to OSI's definition](https://opensource.org/faq#commercial)).

Please don't call things which are not open source as "open source" (according to the most widely accepted definition of "open source").

**Comment by Sameedahmad** [comment]
Hmm, Interesting.

**Comment by RJT1990** [comment]
thanks, changed and will push!

**Comment by Aspie96** [comment]
I agree with OP and am thankful to @RJT1990 for looking into this. I'd like to add my 2 cents.

While it's generally not advisable to use it for software, the "CC BY" license would, in fact, be a free license, and would make the model open source to any reasonable understanding of the definition if it is not restricted by patents (and INAL, but while the CC-BY license doesn't give anyone a patent license, this would still come from the Apache 2.0 license).

For Creative Commons licenses and other non-software the phrase "free cultural" is often used, but the set of basic freedoms is the same as that of the open source definition, the DFSG and the FSD.

I don't know whether the update by @RJT1990 refers to a license change or to a language change, since either could align them. In the latter case, however, I think this resource can be useful to any reader: https://creativecommons.org/share-your-work/public-domain/freeworks/

I believe one of the many great advantages of truly free/open licenses for machine learning models, besides accelerating technological progress, is also the fact that it can reduce the overall amount of energy used. In other words, open source is green-friendly.

To summarize, I agree with the sentiment that "models want to be free". If Papers With Code has a pledge to open sourcing their model, any license that provides that level of freedom would achieve this goal, even if not directly OSI-approved. "CC0", "CC BY" and "CC BY-SA" would all be viable options if Papers With Code finds that, for some reason, a Creative Commons license is preferred.

**Comment by koute** [comment]
> If Papers With Code has a pledge to open sourcing their model, any license that provides that level of freedom would achieve this goal, even if not directly OSI-approved. "CC0", "CC BY" and "CC BY-SA" would all be viable options if Papers With Code finds that, for some reason, a Creative Commons license is preferred.

Indeed. But CC-BY-NC is not, which is what Facebook seems to be releasing all of their models under. Which I think is very unfortunate, since that makes the models effectively useless for anything (depending on the jurisdiction it might be illegal even for other researchers to use them for research, also see [here](https://researchdata.springernature.com/posts/for-open-data-think-twice-before-applying-non-commercial-conditions) for a decent writeup), but it is their work, and it is fair for them to release it under whichever license they want. I just wish the license isn't misrepresented to be what it is not.

**Comment by nealmcb** [comment]
I see that the README.md in the repo no longer claims that the model is open source (except in a prompt...) - thanks.
But the paper still claims that it is, in the abstract no less. Please fix that, and include in the README a clarification of the earlier, widely-reported claims that it was open source. And ideally, explain why you're not actually open sourcing the model. Thanks!

**Comment by sherlcok314159** [comment]
So we still can not use these models for any commercial use. Is that true? @RJT1990
